user-guide/requests-list.md

```diff
@@ -9,51 +9,54 @@ position: 60
 
 # Requests List
 
-The **Requests** list in Fiddler Everywhere provides the option to save, modify, execute, and share previously composed requests. The [Composer](https://docs.telerik.com/fiddler-everywhere/user-guide/composer) tab enables you to build and send HTTP and HTTPS request manually. When you **save** a request using the Composer, the request is added to a **collection** in the **Requests** list. 
+The **Requests** list in Fiddler Everywhere provides the option to save, modify, execute, and share previously composed requests. When you **save** a request using the [Composer](https://docs.telerik.com/fiddler-everywhere/user-guide/composer), the request is added to a **collection** in the **Requests** list.
+
+This article describes how to organize and modify existing requests. To learn more about building and sending HTTP and HTTPS requests, see the [Composer](https://docs.telerik.com/fiddler-everywhere/user-guide/composer) article.
 
 ![Requests list](../images/requests/requests-list-all.png)
 
-The Requests list contains one or more **collections** (root folders) of saved requests (or subfolders). You can also further organize your request in subfolders. To reload or start working on the saved requests, expand a collection, and double click on a selected request. 
+The **Requests** list is organized in **collections**. Each collection is a folder on the root level of the **Requests** list. A collection can contain saved requests and other subfolders. To reload or start working on the saved requests, expand a collection, and double click on a selected request.
```

Comment:
I find the concept of folders and collections confusing, especially when the GUI refers to collections as folders - for example, when creating a new request. If we call the root level folder a _Collection_ and everything under the collection simply a _folder_, this would make things a bit simpler.